plugin/Gui/AdminPage.php

```diff
@@ -43,4 +43,12 @@ public function registerForms()
 			$pInputModelRenderer->registerFields($pFormModel);
 		}
 	}
+
+	public function generateSearchForm($page,$button)
+	{
+		echo '<form action="'.admin_url('admin.php').'" method="get" id="onoffice-form-search">'.$page.
```

Comment:
Please remember to escape values

Reply:
@jayay $page is an input tag. I have already escape the value of this input before.